### PR TITLE
add updated readme

### PR DESCRIPTION
**Pull request by giranezafiacre**

### CODE REVIEW

**Comment by giranezafiacre** [comment]
[Add an Updated Readme File to the project](https://trello.com/c/Odv9SDDi/124-add-an-updated-readme-file-to-the-project)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,80 @@
...
+
+# API Documentation :pencil:
+API endpoints Documentation are hosted [Here](https://phantom-backend.herokuapp.com//swaggerDocument/)
+
+## Technologies && Tools
+
+* [NodeJS](https://nodejs.org/) - JavaScript Runtime Environment
+* [ExpressJs](https://expressjs.com/) - A Minimal  Web Application Framework
+* [postgres](https://www.postgresql.org/) -open source object-relational database system.
+* [jest](https://jestjs.io/) - A JavaScript test framework for Node.js programs, asynchronous testing, test coverage reports, and use of any assertion library
```
Comment (Isaackomeza) [inline README.md]: Thank you @giranezafiacre but here in tools and technology I think you can add other more:

- Docker
- Swagger documentation
- ESLint
- CI/CD Method used in the project

Comment (giranezafiacre) [reply README.md]: - Resolved @Isaackomeza

**Review by fistonhn — COMMENTED** [review]
@giranezafiacre, you have done a great thing. The only one thing you should also add is an acknowledgment.
ack

**Comment by giranezafiacre** [comment]
@fistonhn I will just after a pre-demo, thanks for feedback

**Comment by giranezafiacre** [comment]
thank you @fistonhn, I resolved it

**Pull request merged**